test/integration/cancel_task_test.js

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+import slugid from 'slugid';
+import co from 'co';
+import taskcluster from 'taskcluster-client';
+import DockerWorker from '../dockerworker';
+import TestWorker from '../testworker';
+
+var jsonFromUrl = JSON.parse(fs.readFileSync('test/integration/cancelTaskReference.json'));
+
+suite('Cancel Task', () => {
+  test("cancel", async () => {
+    var CancelQueue = taskcluster.createClient(jsonFromUrl);
```

Comment:
This will be removed once the queue is deployed.  Only for local testing with the queue rewrite.